recipes/librtlsdr/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+required_conan_version = ">=1.28.0"
+
+
+class LibRtlSdrConan(ConanFile):
+    name = "librtlsdr"
+    description = "Software to turn the RTL2832U into an SDR"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://osmocom.org/projects/rtl-sdr/wiki/Rtl-sdr"
+    license = "GPL-2.0"
+    topics = ("conan", "sdr", "rtl-sdr")
+    generators = ("cmake", "cmake_find_package")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
```

Comment:
You forgot to add a `CMakeLists.txt` cmake wrapper script.

Reply:
I actually don't have one haha. Paste fail...

Reply:
I actually don't have one haha. Paste fail...